libraries/reconstruction_cuda/cuda_gpu_geo_transformer.cpp

```diff
@@ -185,7 +185,7 @@ void GeoTransformer<T>::applyBSplineTransform(
         const std::pair<Matrix1D<T>, Matrix1D<T>> &coeffs, size_t imageIdx, T outside) {
     checkRestrictions(3, output, input, coeffs, imageIdx);
 
-    loadOutput(output, outside);
+    // loadOutput(output, outside);
```

Comment:
I don't think this is true (that all pixels will be overwritten anyway). 
This was to ensure that in case the interpolation method would read behind the scope of the input image (i.e. the value would be undefined), we would have default value there.